Sources/PointFree/Mailgun/SendEmail.swift

```diff
@@ -0,0 +1,56 @@
+import Either
+import Html
+import Prelude
+
+func sendEmail(
+  from: String,
+  to: [String],
+  subject: String,
+  content: Either3<String, [Node], (String, [Node])>,
+  domain: String = "mg.pointfree.co"
+  )
+  -> EitherIO<Unit, SendEmailResponse> {
+
+    let (plain, html): (String, String?) =
+      destructure(
+        content,
+        { plain in (plain, nil) },
+        { nodes in (plainText(for: nodes), render(nodes)) },
+        second(render)
+    )
+
+    return AppEnvironment.current.sendEmail(
+      Email(
+        from: from,
+        to: to,
+        cc: nil,
+        bcc: nil,
+        subject: subject,
+        text: plain,
+        html: html,
+        testMode: nil,
+        tracking: nil,
+        trackingClicks: nil,
+        trackingOpens: nil,
+        domain: domain
+      )
+    )
+}
+
+// TODO: move to swift-prelude
+private func destructure<A, B, C, D>(
```

Comment:
I have a branch on prelude for making a bunch of these, but I also went down a wild rabbit hole that I need to dig myself out of